README.md

```diff
@@ -25,9 +25,9 @@ The following dependencies are required and must be installed separately.
 
 * **[FFmpeg](https://www.ffmpeg.org)**
 
-> **Note**: FFmpeg does not affect the download, only affect the final file merge.
+> **Note**: FFmpeg does not affect the download, only affects the final file merge.
 
-To install Annie, please use `go get`, or download the binary file in the [Releases](https://github.com/iawia002/annie/releases) page, or compile yourself.
+To install Annie, please use `go get`, download the binary file in the [Releases](https://github.com/iawia002/annie/releases) page, or compile yourself.
```

Comment:
why remove `or`?

Reply:
When listing more than two objects or actions, 'or' is only needed for the last object/action. It's a grammar rule and makes it easier to read. It's just a small quirk.

Reply:
Ok then